example/objc/main_darwin.go

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2022 The Ebitengine Authors
+
+//go:build example
+
+package main
+
+import (
+	"fmt"
+	"unsafe"
+
+	"github.com/ebitengine/purego/objc"
```

Comment:
As this is imported here, do we need `example/main.go`?

Reply:
Wasn't that to test that purego compiled on Darwin and Windows? This only works on Darwin bc of ObjC

Reply:
I'm suggesting to remove `example/main.go`

Reply:
Or, did you mean `example/main.go` is still needed for Windows?

Reply:
Yeah that's what I was saying. I believe it was added for Windows since purego didn't compile correctly on Windows and so the example was created to ensure that it was.

Reply:
Sure!